sui_types/src/messages.rs

```diff
@@ -334,14 +334,29 @@ const_assert_eq!(
 
 */
 
-/// An transaction signed by a single authority
 #[derive(Debug, Eq, Clone, Serialize, Deserialize)]
-pub struct SignedTransaction {
-    pub transaction: Transaction,
+pub struct AuthoritySignedData<D> {
```

Comment:
Good idea to have one type for this. A couple of observations:
- Should we have a uniform way to convert these to certificates,
- Should we have a uniform authority certificate type too?